src/data/duckbot_train_data.csv

```diff
@@ -0,0 +1,13 @@
+How can I become a member?,"The club membership costs $10 for the full year. You can pay for the membership online on our website here: https://csclub.org.au/join/ Alternatively, you can pay at any upcoming club event or contact one of the committee members here!"
+What benefits do I get with the membership?,"The membership gives you access to The Duck Lounge (located at Engineering and Maths EM110), educational workshops, social nights, and a wide network of other computer science students and graduates. "
+What and where is the Duck Lounge? ,"The Duck Lounge is located at EM110 in the Engineering and Maths building. It has a study space with power outlets and couches, along with a Nintendo Switch for games. We also sell snacks and drinks at affordable prices."
+What is Ravi's Study Program (RSP)?,"Participating in RSP (Ravi's Study Program) could possibly be the single most beneficial thing you can do while you're at University to land a job as a Software Engineer at a great company. The program is run by current and former students and tutors from The University of Adelaide, and provides students with the exact steps they need to take to prepare themselves to pass interviews at big tech/FAANG companies. There are very impressive numbers of people that land internships at these companies after successfully completing the program."
+How can I get involved with/join RSP?,"RSP is an intensive programming bootcamp that runs each summer break. If you wish to get involved, keep an eye out for flyers on campus towards the end of Semester 2 and advertisements that are posted in the Discord."
+Who are you?,"I'm duckbot, the official discord bot for the University of Adelaide CS Club Discord. I'm here to help you know more about the CS club, upcoming events, and more!"
+CS club membership join link,"https://csclub.org.au/join/"
+CS club info,"The University of Adelaide Computer Science Club is a student-run club for those with an interest in computer science or computing in general. Although we're a university club, we welcome anyone interested in computer science and/or socialising to join! The club has frequent events such as game nights, computer science talks, educational workshops, and programming competitions."
+What is the Competitive Programming Club (CPC)?,"The Competitive Programming Club (CPC) is a student-run club dedicated to promote and nurture the sport of competitive programming at the University of Adelaide. The club hosts frequent practice competitions and training workshops, discusses algorithms, and celebrate with free food. To learn more about it, visit https://acpc.io/."
+Competitive Programming Club (CPC) home page, "https://acpc.io/"
+CPC link, "https://acpc.io/" 
+CS Club link, "https://csclub.org.au/"
```

Comment:
Some changes/additions to make the answers more accurate. We might add some more later on in a seperate PR.

```suggestion
What is the Computer Science (CS) Club?,"The University of Adelaide Computer Science Club is a student-run club for those with an interest in computer science or computing in general. Although we're a university club, we welcome anyone interested in computer science and/or socialising to join! The club has frequent events such as game nights, computer science talks, educational workshops, and programming competitions. To learn more about the club, visit https://csclub.org.au/."
How can I become a member?,"The club membership costs $10 for the full year. You can pay for the membership online on our website here: https://csclub.org.au/join/ Alternatively, you can pay at any upcoming club event or contact one of the committee members here!"
What benefits do I get with the membership?,"The membership gives you access to The Duck Lounge (located at Engineering and Maths EM110), educational workshops, social nights, club resource drive, and a wide network of other computer science students and graduates. "
What and where is the Duck Lounge? ,"The Duck Lounge is located at EM110 in the Engineering and Maths building. It has a study space with power outlets and tables, along with a Nintendo Switch for games. We also sell snacks and drinks at affordable prices."
What is Ravi's Study Program (RSP)?,"Participating in RSP (Ravi's Study Program) could possibly be the single most beneficial thing you can do while you're at University to land a job as a Software Engineer at a great company. The program is run by current and former students and tutors from The University of Adelaide, and provides students with the exact steps they need to take to prepare themselves to pass interviews at big tech/FAANG companies. There are very impressive numbers of people that land internships at these companies after successfully completing the program."
How can I get involved with/join RSP?,"RSP is an intensive programming bootcamp that runs each summer break. If you wish to get involved, keep an eye out for flyers on campus towards the end of Semester 2 and advertisements that are posted in the Discord."
Who are you?,"I'm DuckBot, the official Discord bot for the University of Adelaide CS Club Discord. I'm here to help you know more about the CS Club, upcoming events, and more!"
CS Club info,"The University of Adelaide Computer Science Club is a student-run club for those with an interest in computer science or computing in general. Although we're a university club, we welcome anyone interested in computer science and/or socialising to join! The club has frequent events such as game nights, computer science talks, educational workshops, and programming competitions."
What is the Competitive Programming Club (CPC)?,"The Competitive Programming Club (CPC) is a student-run club dedicated to promote and nurture the sport of competitive programming at the University of Adelaide. The club hosts frequent practice competitions and training workshops, discusses algorithms, and celebrate with free food. To learn more about it, visit https://acpc.io/."
What are Friday Night Games, "Friday Night Games (FNG) is a regular Computer Science Club event where the Duck Lounge (Located in EM110) hosts a weekly games night from 5PM where members commonly play amongst a Nintendo Switch and board games. Moreover, on the final Friday of each month within semester, we have free food for all members! Type ``/faq fng food`` to find out when the next FNG with food will be. Join us for games and fun!" 
Who is the Open Source Team?, "The CS Club Open Source Team is a place for creative and inventive students who want to build innovative open-source software together. We are a new team currently working on a few projects, but we have many more exciting projects planned for the future!" 
How to join Open Source Team?, "Looking to join an active and welcoming student developer community? We're always on the lookout for enthusiastic and creative students to join us, learn, and contribute to making useful open-source projects. Join our team by filling out https://forms.gle/rwkByjjhyZ8Q6gea6." 
What our our Open Source Team projects?, "Our projects include the CS Club Website, DuckBot, Voxel Game Engine + Game, Courses API, MyTimetable, MyStudyPlan, and MyCourseReviews. Learn more about them and the Open Source Team on our website: https://www.csclub.org.au/open-source" 
CS Club link, "https://csclub.org.au/"
CS Club membership join link,"https://csclub.org.au/join/"
Open Source Team info link, "https://www.csclub.org.au/open-source"
Open Source Team signup link, "https://forms.gle/rwkByjjhyZ8Q6gea6"
CPC link, "https://acpc.io/" 
```